heft-plugins/heft-jest-plugin/src/jestPlugin.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Heft Jest Plugin Options Configuration",
+  "description": "Defines additional Jest-related plugin configuration.",
+  "type": "object",
+
+  "additionalProperties": false,
+
+  "properties": {
+    "disablePresetRelativeModuleResolution": {
```

Comment:
I made this enabled by default, mostly due to the fact that the included `jest-shared.config.json` depends on it being enabled, otherwise the config will fail to resolve. Thoughts on making this default-on given that it is non-standard and will break Jest configs that break the package boundary?

Reply:
The latter :) I want to enable the feature by default, but defaulting to `true` felt wrong, so I defaulted to `false` and named it this. Though, clearly it's still confusing...

Reply:
In switching to heft-config-file, I've replaced the preset property with `extends`. This is non-standard and not supported by Jest. However, so is `node_modules` resolution in Jest configs. So it becomes an implicit "I understand that this is not Jest supported", which I like.

Reply:
Alright swapped over to heft-config-file. Left the option in but renamed it and default to "true" internally